src/app/rosetta/rosetta-cli-config/mina-no-delegation-test.ros

```diff
@@ -1,11 +1,10 @@
-// This is the version of the Construction API tests does not test
-// stake delegation, which cannot be tested with rosetta-sdk-go as
-// of v. 0.8.2.
-
 // Workflow used to create new accounts
 create_account(1){
   create{
-    network = {"network":"debug", "blockchain":"mina"};
+    network = {
+      "network":"berkeley-ITN",
```

Comment:
Yes, it should. That is a leftover from my ITN tests. In the future, we can replace these placeholder values by loading them from environment variables and adjust the integration test scripts to set these env variables instead of searching and replacing them on the file.